docs/configuration.md

```diff
@@ -141,17 +141,30 @@ analyzers:
 
 ### Add a Custom Message to the Posted Comments
 
-You can configure **source{d} Lookout** to add a custom message to every comment that each analyzer returns. This custom message will be created following the rule:
-```
-sprintf(providers.github.comment_footer, feedback)
-```
-If any of those two keys are not defined, the custom message won't be added.
+You can configure **source{d} Lookout** to add a custom message to every comment that each analyzer returns. This custom message will be created from the template defined by `providers.github.comment_footer`, using the `name` and `feedback` set for each analyzer.
 
-Example:
-```text
-"_If you have feedback about this comment, please, [tell us](%s)._"
+If the template (`providers.github.comment_footer`) is empty, or the analyzer configuration does not define any of the values that the template requires, the custom message won't be added.
+
+For example, for this configuration, each analyzer needs to define `name` and `feedback`:
+
+```yaml
+providers:
+  github:
+    comment_footer: "Comment made by analyzer {{.Name}}, [report]({{.Feedback}})."
```

Comment:
Could the maintainer of multiple analyzers be willing to have a different footer message for each analyzer? Additionally the maintainer could also want to have different message for different providers for the same analyzer. Anyway I don't consider this as blocking, but just something that we could consider if this scenario is possible. Who's the responsible of choosing the footer message?

Reply:
Currently, the footer template is defined by `lookoutd` administrator.
I also agree that it should be (somehow) up to the analyzer maintainer, but I think it will come at some point as a feature request from @src-d/product